<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>generate_new_token</class>
 <widget class="QDialog" name="generate_new_token">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Usuario:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="user_line">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="generate_button">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Generar token</string>
   </property>
  </widget>
  <widget class="QPushButton" name="token_1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="token_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="token_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="token_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="token_5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="token_6">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nuevo token:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Contraseña</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>de confirmación:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_line">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="generate_button_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar cambios</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
